src/Codeception/Test/Loader.php

```diff
@@ -34,13 +44,18 @@
  * $testLoader = new \Codeception\TestLoader('tests/unit');
  * $testLoader->loadTest('models'); // all tests from tests/unit/models
  * $tests = $testLoader->getTests();
- * ?>
  * ```
  *
  */
 class Loader
 {
+    /**
+     * @var Cept[]|Cest[]|Gherkin[]|Unit[]
```

Comment:
It is `@var LoaderInterface[]`

The code in constructor supports adding custom loaders.